### PR TITLE
Viktor

### PR DESCRIPTION
**Pull request by pvvArienBy**
add CurrencyCreateDTO
       CurrencyDTO

### CODE REVIEW

**Review by Pavel-Kazlou — APPROVED** [review]
approve

File: src/main/java/by/it_academy/jd2/Mk_JD2_98_23/core/dto/CurrencyCreateDTO.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
+package by.it_academy.jd2.Mk_JD2_98_23.core.dto;
+
+import java.time.LocalDateTime;
+
+public class CurrencyCreateDTO {
+
```
Comment (Dzmitry-Highlander) [inline src/main/java/by/it_academy/jd2/Mk_JD2_98_23/core/dto/CurrencyCreateDTO.java]: add constructor

File: src/main/java/by/it_academy/jd2/Mk_JD2_98_23/core/dto/CurrencyDTO.java [hunk 2]
```diff
@@ -1,4 +1,163 @@
...
+    private String Cur_Name_BelMulti;
+    private String Cur_Name_EngMulti;
+    private int Cur_Scale;
+    private int Cur_Periodicity;
+    private LocalDateTime Cur_DateStart;
+    private LocalDateTime Cur_DateEnd;
+
+    public CurrencyDTO() {
+    }
+
```
Comment (Dzmitry-Highlander) [inline src/main/java/by/it_academy/jd2/Mk_JD2_98_23/core/dto/CurrencyDTO.java:27]: add constructor

**Review by Dzmitry-Highlander — APPROVED** [review]
good

**Pull request merged**